GentWebApi/Models/RegisterModel.cs

```diff
@@ -1,15 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
+﻿
 namespace GentWebApi.Models {
 	public class RegisterModel {
```

Comment:
RegisterModel nodig? als je gwn [Required] bij attributen zet is een aparte klasse voor register niet nodig volgens mij

Reply:
kvind da duidelijker en tis ook een beetje meer secure haha. tis een dto he :p

Reply:
ahja ok